Dockerfile

```diff
@@ -15,12 +15,10 @@ RUN mkdir -p bin &&\
   LDFLAGS="$LDFLAGS -X github.com/go-swagger/go-swagger/cmd/swagger/commands.Version=${tag_name}" &&\
   CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -tags osusergo,netgo -o bin/swagger -ldflags "$LDFLAGS" -a ./cmd/swagger
 
-FROM --platform=$TARGETPLATFORM golang:alpine
+FROM --platform=$TARGETPLATFORM alpine
 
 LABEL maintainer="Ivan Porto Carrero <ivan@flanders.co.nz> (@casualjim)"
 
-RUN apk --no-cache add ca-certificates shared-mime-info mailcap git build-base
```

Comment:
ca-certificates, shared-mime-info and mailcap would still be required otherwise not all HTTP functionality will work